Yona/BuildOutput/ExportOptionsRelease.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same here: Can't we move this file outside the `BuildOutput` directory? It is not build output but build input...

Reply:
we've moved this out of the build output folder. We have added a Export options folder in GIT to store the plist's for each configuration